src/game/server/ddracecommands.cpp

```diff
@@ -91,9 +91,13 @@ void CGameContext::ConKillPlayer(IConsole::IResult *pResult, void *pUserData)
 	{
 		pSelf->m_apPlayers[Victim]->KillCharacter(WEAPON_GAME);
 		char aBuf[512];
-		str_format(aBuf, sizeof(aBuf), "%s was killed by %s",
-			pSelf->Server()->ClientName(Victim),
-			pSelf->Server()->ClientName(pResult->m_ClientId));
+		if(pResult->NumArguments() == 2)
+			str_format(aBuf, sizeof(aBuf), "%s was killed by authorized user (%s)",
```

Comment:
Same as in `force_vote` command
```suggestion
			str_format(aBuf, sizeof(aBuf), "%s was killed by authorized player (%s)",
```